plugins/source/aws/resources/services/ssoadmin/instances_mock_test.go

```diff
@@ -16,6 +16,7 @@ func buildInstances(t *testing.T, ctrl *gomock.Controller) client.Services {
 	im := types.InstanceMetadata{}
 	ps := types.PermissionSet{}
 	as := types.AccountAssignment{}
+	ip := "{\"key\": \"value\"}"
```

Comment:
```suggestion
	ip := `{"key": "value"}`
```

Also a nit, but using backticks for the string allows you to do away with the backslashes